src/account_management.rs

```diff
@@ -0,0 +1,163 @@
+        Manager::link_secondary_device(store, SignalServers::Production, device_name, tx),
+        async move {
+            match rx.await {
+                Ok(url) => {
+                    println!("Scan the QR code to link the device to account '{}'!", name);
+                    qr2term::print_qr(url.as_ref()).expect("QR generation failed");
+                    println!("You can also use the URL: {url}");
+                }
+                Err(err) => println!("Error while linking device: {err}"),
+            }
+        },
+    )
+    .await;
+
+    let mut manager = match manager_result {
+        Ok(manager) => {
+            println!("Device linked successfully! Syncing contacts...");
+            manager
+        }
+        Err(e) => return Err(anyhow!("Error while linking device: {e}")),
+    };
+
+    initial_sync(&mut manager).await?;
+    let mut config = Config::load();
+    config.set_current_account(account_name.clone());
+    config
+        .save()
+        .map_err(|e| anyhow::anyhow!("Failed to save config: {}", e))?;
+
+    println!("Account '{}' created and set as current!", account_name);
+    Ok(())
+}
+
+pub async fn list_accounts_cli() -> Result<()> {
+    let accounts = list_accounts()?;
+    let config = Config::load();
+    let current = config.get_current_account();
+
+    if accounts.is_empty() {
+        println!("No accounts found.");
+        return Ok(());
+    }
+
+    println!("Available accounts:");
+    for account in accounts {
+        let marker = if current == Some(&account) {
+            " (current)"
+        } else {
+            ""
+        };
+        println!("  - {}{}", account, marker);
+    }
+    Ok(())
+}
+
+pub async fn switch_account_cli(account_name: String) -> Result<()> {
+    let accounts = list_accounts()?;
+    if !accounts.contains(&account_name) {
+        return Err(anyhow::anyhow!("Account '{}' does not exist", account_name));
+    }
+
+    let mut config = Config::load();
+    config.set_current_account(account_name.clone());
+    config
+        .save()
+        .map_err(|e| anyhow::anyhow!("Failed to save config: {}", e))?;
+
+    println!("Switched to account '{}'", account_name);
+    Ok(())
+}
+
+pub async fn get_current_account_cli() -> Result<()> {
+    let config = Config::load();
+    match config.get_current_account() {
+        Some(account) => println!("Current account: {}", account),
+        None => println!("No current account set"),
+    }
+    Ok(())
+}
+
+pub async fn delete_account_cli(account_name: String) -> Result<()> {
+    use std::io::{self, Write};
+
+    let accounts = list_accounts()?;
+    if !accounts.contains(&account_name) {
+        return Err(anyhow::anyhow!("Account '{}' does not exist", account_name));
+    }
+
+    let config = Config::load();
+    let is_current = config.get_current_account() == Some(&account_name);
+
+    print!(
+        "Are you sure you want to delete account '{}'? This action cannot be undone! (yes/no): ",
```

Comment:
```suggestion
    print!(
        "Are you sure you want to delete account '{}'? This action cannot be undone! [y/N]: ",
```

Make "No" default for empty input, make that only inputs 'y' or 'Y' delete account and put that inside of a loop so every other input results in asking for confirmation again.